app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> int:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int) -> int:
```

Comment:
fix it and other places 

```suggestion
    def __add__(self, other: int | float | Distance) -> Distance:
```

Reply:
коли використовую  def __add__(self, other: int | float | Distance) -> Distance:
при перевірці виходить помилка  
test_main.py:None (test_main.py)
ImportError while importing test module 'C:\Olexandr\py-distance-class\tests\test_main.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
..\venv\lib\site-packages\_pytest\python.py:608: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
..\venv\lib\site-packages\_pytest\pathlib.py:533: in import_path
    importlib.import_module(module_name)
C:\Users\1\AppData\Local\Programs\Python\Python39\lib\importlib\__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1030: in _gcd_import
    ???
<frozen importlib._bootstrap>:1007: in _find_and_load
    ???
<frozen importlib._bootstrap>:986: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:680: in _load_unlocked
    ???
..\venv\lib\site-packages\_pytest\assertion\rewrite.py:168: in exec_module
    exec(co, module.__dict__)
test_main.py:6: in <module>
    from app.main import Distance
..\app\main.py:1: in <module>
    import Distance
E   ModuleNotFoundError: No module named 'Distance'
collected 0 items / 1 error
спробував інсталювати pip install Distance
Requirement already satisfied: Distance in c:\olexandr\py-distance-class\venv\lib\site-packages (0.1.3)
помилка не зникає, що робити?

Reply:
коли використовую def add(self, other: int | float | Distance) -> Distance:
при перевірці виходить помилка
test_main.py:None (test_main.py)
ImportError while importing test module 'C:\Olexandr\py-distance-class\tests\test_main.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
..\venv\lib\site-packages_pytest\python.py:608: in importtestmodule
mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
..\venv\lib\site-packages_pytest\pathlib.py:533: in import_path
importlib.import_module(module_name)
C:\Users\1\AppData\Local\Programs\Python\Python39\lib\importlib_init.py:127: in import_module
return _bootstrap._gcd_import(name[level:], package, level)
:1030: in _gcd_import
???
:1007: in _find_and_load
???
:986: in _find_and_load_unlocked
???
:680: in _load_unlocked
???
..\venv\lib\site-packages_pytest\assertion\rewrite.py:168: in exec_module
exec(co, module.dict)
test_main.py:6: in
from app.main import Distance
..\app\main.py:1: in
import Distance
E ModuleNotFoundError: No module named 'Distance'
collected 0 items / 1 error
спробував інсталювати pip install Distance
Requirement already satisfied: Distance in c:\olexandr\py-distance-class\venv\lib\site-packages (0.1.3)
помилка не зникає, що робити?